mm2src/coins/nft.rs

```diff
@@ -1467,9 +1580,11 @@ pub(crate) fn get_domain_from_url(url: Option<&str>) -> Option<String> {
 
 /// Clears NFT data from the database for specified chains.
 pub async fn clear_nft_db(ctx: MmArc, req: ClearNftDbReq) -> MmResult<(), ClearNftDbError> {
+    // TODO: db_id
+    let db_id: Option<String> = None;
```

Comment:
This easy to fix I think, we should use the enabled platform coin `db_id`. Similar to here https://github.com/KomodoPlatform/komodo-defi-framework/blob/24774a46da48a40c218a724ff580578badcbef2d/mm2src/coins/nft.rs#L1656-L1657
I think you should start implementing `account_db_id` for all coins, this is needed for hd wallet (in utxo and eth) where the `db_id` will be the `enabled_address` one.

Reply:
c.c. @laruh

Reply:
> This easy to fix I think, we should use the enabled platform coin db_id

Speaking in general (not about should we use _platform_ coin db_id or id of _enabled_address_): if to fix this in `clear_nft_db` you need to require Nft activation, then please do it. Its not a problem, since most of NFT RPC already ask about it.

> I think you should start implementing account_db_id for all coins, this is needed for hd wallet (in utxo and eth) where the db_id will be the enabled_address one.

need to do a deeper review of how current multi-pubkey feature work, to answer this question. 
But now my answer is we should treat global NFT the same way as erc20 token (in general implementations), it is just not registered in`erc20_tokens_infos` platform coin field. So nft behavior should not interfere with the implementation of suggested idea.

Reply:
done

Reply:
I guess you forgot to use `db_id` here
https://github.com/KomodoPlatform/komodo-defi-framework/blob/916ce6a33dfa2320ed2809674d217c08fceed6c3/mm2src/coins/nft.rs#L1590